test/integration.js

```diff
@@ -270,7 +261,7 @@ async function testRejectState(t, expectedReason, makeNewState) {
 tape('RejectState: wrong signature (InvalidSignature)', async function(t) {
 	await testRejectState(t, 'InvalidSignature', function(newState) {
 		// increase the balance, so we effectively end up with a new state
-		const balances = { ...newState.balances, someoneElse: '1' }
+		const balances = { ...newState.balances, '0x033ed90e0fec3f3ea1c9b005c724d704501e0196': '1' }
```

Comment:
again, consider putting this in the `dummyVals` but I have an important question here:

how do we make sure that `0x033` does not override something that's already in `newState.balances`? 

also, in the second case, how do we ensure that '0x033' was not previously present in the `newState.balances`
how about generating a random addr in the second case?

Reply:
I see now that this is the `creator` from dummyVals

I think that you should indeed gen a random addr here, in this test and in the next (`const fakeBalances=`)

Reply:
Yeah true, I have added random address generation